examples/gallery/symbols/basic_symbols.py

```diff
@@ -0,0 +1,117 @@
+"""
+Basic geometric symbols
+-----------------------
+
+The :meth:`pygmt.Figure.plot` method can plot individual geometric symbols
+by passing the corresponding shortcuts to the ``style`` parameter. The 14 basic
+geometric symbols are shown underneath their corresponding shortcut codes.
+"""
+
+import pygmt
+
+fig = pygmt.Figure()
+fig.basemap(region=[0, 8, 0, 3], projection="X12c/4c", frame=True)
+
+# define fontstlye for annotations
+font = "15p,Helvetica-Bold"
+
+# upper row
+y = 2
+
+# use a dash in x direction (-) with a size of 0.9 cm,
+# linewidth is set to 2p and the linecolor to "gray40"
```

Comment:
The `pen` and `color` parameters are used in all those symbols. How about explaining them in the docstrings instead of the codes?

Reply:
You mean to explain the usage og ``pen`` and ``color``  @core-man ?

Reply:
> You mean to explain the usage of `pen` and `color` @core-man ?

Yes, explain them in the docstrings, while removing the comments in the code?

Reply:
I added an explanation in the docstring, however, I would leave the comments in the code since the parameter values slightly differ for some cases.